devito/propagator.py

```diff
@@ -119,6 +117,11 @@ def run(self, args):
 
         f = self.cfunction
 
+        # appends block sizes if cache blocking
+        for block in self.block_sizes:
+            if block is not None:
+                args.append(block)
```

Comment:
that is:
``args += [block for block in self.block_sizes if block]``
or (no creation of new lists)
``args.extend([...])``
?